Taskfile.yml

```diff
@@ -194,3 +194,94 @@ tasks:
   build-all-images:
     desc: Build all container images (main app and egress proxy)
     deps: [build-image, build-egress-proxy]
+
+  # Vault Agent Injector Integration Tasks
+  kind-with-vault:
+    desc: Setup Kind cluster with ToolHive operator and Vault Agent Injector
+    cmds:
+      - task: kind-with-toolhive-operator-local
```

Comment:
lmk if the amount of task targets is getting too big and this should be a script instead. I found having a reproducable and scriptable way of setting up Vault super useful for me, though. It's also living docs

Reply:
Yeah, I think it did reach a script point

Reply:
it's a script now!